File: 240407/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
arr2 = ["apple", "banana", "grape", "blueberry", "orange"]

n = input()
cnt = 0

for i in range(5):
    c = False
    for j in range(2,4):
        if n==arr2[i][j]:
            c = True
    if c==True:
        print(arr2[i])
        cnt+=1
print(cnt)

File: 241101/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char a = sc.next().charAt(0);
        int cnt = 0;

        String[] arr = new String[]{"apple", "banana", "grape", "blueberry", "orange"};

        for(int i=0; i<arr.length; i++){
            String str = arr[i];
            for(int j =2; j<4; j++){
                if(a == str.charAt(j)){
                    System.out.println(arr[i]);
                    cnt++;
                    break;
                }
            }
        }
        System.out.println(cnt);
        // 여기에 코드를 작성해주세요.
    }
}